### DISCUSSION TITLE
Setting up Organization Github account

### DISCUSSION THREAD

**Discussion by rajib-iotian** [community/community#42657] [New to GitHub]
Hello,

I would like to setup Github account for my team. What are the options do I have so that I can select the right option with right price. 

Also, I am new to Github where can I find documents and training videos about the Github usage.

Regards,
Rajib

**Comment by DSDmark** [comment]
GitHub offers several options for creating an account for your team:

  -  GitHub Team: This is a subscription service that provides your team with access to private repositories, team-based permissions, and integrations with tools like Slack and Jira. It also includes features like code review and project management. You can purchase a GitHub Team subscription for $4 per user per month.

   - GitHub Enterprise: This is a self-hosted version of GitHub that you can install on your own servers. It includes all the features of GitHub Team, as well as additional security and compliance features. GitHub Enterprise is priced on a per-seat basis, with prices starting at $21 per user per month.

   - GitHub Free: This option provides your team with access to public repositories, as well as some limited collaboration features like issue tracking and pull requests. It's a good option if you're just getting started with GitHub and want to try out the platform before committing to a paid subscription.

You can find everything's there:- [**GitHub docs**](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations#about-organizations)

**Reply by rajib-iotian** [reply]
Thank you for providing basic information.

If I used Github Free version, if I check-in the code then it will be visible to public. Since the code is meant for internal purpose, can I protect it using Github Free?

From where do I need to place order for Github Team. Can you please share the link? Where can I find training videos or documents on the how to use Team.

**Reply by DSDmark** [reply]
Hi @rajib-iotian,
Yes, if you use the free version of GitHub, your code will be visible to the public. If you want to protect your code and keep it private.

To purchase GitHub Team, you can visit the pricing page on the GitHub website: https://github.com/pricing

On the pricing page, you can select the GitHub Team plan and follow the steps to sign up and purchase the plan.

To learn how to use GitHub Team, you can refer to the GitHub documentation and tutorials, which are available on the GitHub Help page: https://help.github.com/

There are also a number of resources available online that can help you get started with GitHub and learn how to use it effectively. Some popular options include video tutorials on YouTube, online courses and tutorials, and blog posts and articles on websites like Medium and dev.to.
